### DISCUSSION TITLE
Developer mode on Chrome can alter how the filter list is used?

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#3239] [General]
Turn on Developer mode and enable EasyList/uBO Cookie Notices. It will be 51,972 used out of about more than 68k filters I can't remember exactly. On another Chrome profile which Developer mode is turned off. Enabling EasyList/uBO Cookie Notices will be 51,972 used out of 52,158.

Click to update the filter list manually doesn't restore it to 51,972 used out of 52,158. So I removed uBO on the former profile. Reinstall uBO and enable EasyList/uBO Cookie Notices. Now it's the same 51,972 used out of 52,158.

**Comment by garry-ut99** [comment]
Maybe enabling dev mode is unrelated and is only a random coincidence,
(also most likely unrelated to a browser, because I tested on Firefox):

From uBO's [Dashboard](https://github.com/gorhill/uBlock/wiki/Dashboard) > About pane

<details>
<summary>SCREENSHOOT</summary>

![SCREENSHOOT](https://github.com/uBlockOrigin/uBlock-issues/assets/72945564/37853fe2-8e7e-4a56-b268-65ce17b64327)

</details>

> uBO's own filter lists are freely hosted on the following [CDNs](https://wikipedia.org/wiki/Content_delivery_network):
> [Cloudflare Pages](https://pages.cloudflare.com/), [GitHub Pages](https://pages.github.com/), [jsDelivr](https://www.jsdelivr.com/), [Statically](https://statically.io/)
> A randomly picked CDN is used when a filter list needs to be updated.

When fetched from one of these, uBO shows 52000+:
`https://ublockorigin.github.io/uAssetsCDN/thirdparties/easylist-cookies.txt?_=9`
`https://ublockorigin.pages.dev/thirdparties/easylist-cookies.txt?_=9`


When fetched from this one, uBO shows 68000+:
`https://secure.fanboy.co.nz/fanboy-cookiemonster_ubo.txt?_=9`
because it additionaly includes a sublist which contains additional 16000+:
`https://secure.fanboy.co.nz/easylist_cookie_specific_uBO.txt?_=9`

I'm not sure whether it's a CDN cache/time issue or something related to uBO.

**Reply by gorhill** [reply]
> because it additionaly includes a sublist which contains additional 16000+:
> `https://secure.fanboy.co.nz/easylist_cookie_specific_uBO.txt?_=9`

That list is included in _EasyList – Cookie Notices_, see <https://github.com/uBlockOrigin/uAssets/blob/master/templates/easylist-cookies.template>.

---

`fanboy.co.nz`'s _Easylist Cookie List_ includes twice the content of `https://secure.fanboy.co.nz/easylist_cookie_specific_uBO.txt`, so uBO ends up discarded all the duplicated filters.

**Reply by garry-ut99** [reply]
By the way I've also noticed that `fanboy.co.nz` also includes twice `easylist_cookie_international_specific_hide.txt` header, in fact the second one should be `easylist_cookie_international_specific_block.txt`, but it's only a typo, not a real problem, because in fact only header names are duplicated, not filters, because the second instance of `easylist_cookie_international_specific_hide.txt` header provides filters from `easylist_cookie_international_specific_block.txt`.
